api/api.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"os"
+	"github.com/cloud-native-toolkit/itzcli/pkg"
+	"github.com/cloud-native-toolkit/itzcli/pkg/auth"
+	"github.com/gin-gonic/gin"
+	logger "github.com/sirupsen/logrus"
+)
+
+func StartServer() {
+	r := SetUpRouter()
+	r.Run("localhost:8080")
```

Comment:
8080 is such a common port here that I would be worried about conflicts. Can this be configurable? Or, do we need to have some exception handling here with a nice message that tells the user to close whatever they have running on port 8080 to they can run `itz`?